pages/article/[pid].tsx

```diff
@@ -1,14 +1,48 @@
 import marked from "marked";
-import { useRouter } from "next/router";
+import Router, { useRouter } from "next/router";
```

Comment:
Make sure that the app builds correctly with `npm run build`. If there is an error, fix the error and commit it again